src/components/Navbar.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import styled from 'styled-components'
+import { P } from './Typography'
+import icon from '../assets/nwhacks_icon.svg'
+import { Button } from './Input'
+
+const NavContainer = styled.div`
+  direction: rtl;
+  position: fixed;
+  display: flex;
+  box-sizing: border-box;
+  width: 100vw;
+  height: 5em;
+  padding: 3vh 10vw;
+  margin-right: auto;
+  align-items: center;
+  background: transparent;
+  backdrop-filter: blur(5px);
```

Comment:
wait whats this for?

Reply:
I THINK it's the pic above? like it blurs
![image](https://user-images.githubusercontent.com/3856856/100692917-b145d780-3340-11eb-9b96-5b9a4afa1287.png)

THANK YOU @panjenny0 :))) !!!!

Reply:
is that part of the designs? I didn't notice it

Reply:
I just added the blurring because if it's transparent, the greeting will just blend in with the background
<img width="692" alt="Screen Shot 2020-11-30 at 8 25 35 PM" src="https://user-images.githubusercontent.com/38872354/100697220-37b2e700-334a-11eb-98df-821c4ac3c2b9.png">
<img width="680" alt="Screen Shot 2020-11-30 at 8 25 53 PM" src="https://user-images.githubusercontent.com/38872354/100697235-43061280-334a-11eb-94b0-70764796d3c0.png">
<img width="922" alt="Screen Shot 2020-11-30 at 8 26 10 PM" src="https://user-images.githubusercontent.com/38872354/100697246-4d281100-334a-11eb-8639-bb1774faaef8.png">